docs/ub_theme/SUBTREE.md

```diff
@@ -0,0 +1,2 @@
+The ub_theme folder is a reference to the [ub_theme project](https://github.com/useblocks/ub_theme) - do not edit.
+Instead, you can send your changes to the ub_theme project.
```

Comment:
Can we add here the needed command to update the data in this folder via git-subtree?

Reply:
Yeah sure

Reply:
Mmh, still can't see the command in this file to update the subtree.

Reply:
Oh okay. I added the command to the README.md file instead.